RSSDP/SsdpCommunicationsServer.cs

```diff
@@ -33,7 +33,7 @@ public sealed class SsdpCommunicationsServer : DisposableManagedObjectBase, ISsd
          */
 
         private object _BroadcastListenSocketSynchroniser = new object();
-        private ISocket _BroadcastListenSocket;
+        private List<ISocket> _BroadcastListenSockets;
```

Comment:
Changes to this project is not ideal. We would rather use upstream than make even more changes... but if it can't be avoided?

Reply:
I totally agree with that but the main problem here is that SSDP ignores whatever we set as bind interfaces and just listens on all available IPs. That's not what we should do if we provide means to set those IPs explicitly.